### DISCUSSION TITLE
Failed to load a solhint's config file

### DISCUSSION THREAD

**Discussion by nshen** [nvimtools/none-ls.nvim#80] [Q&A] [answered]
There is an error when I use `{ null_ls.builtins.diagnostics.solhint }`

<img width="870" alt="image" src="https://github.com/nvimtools/none-ls.nvim/assets/181506/66312099-e7a2-4b9c-97a9-fce66c0a4988">

I have a `.solhint.json` on my project root, and I can run solhint on the command line

<img width="758" alt="image" src="https://github.com/nvimtools/none-ls.nvim/assets/181506/046084c1-0e10-419f-aaed-d76ce188d9b7">

Is this a bug relative to https://github.com/jose-elias-alvarez/null-ls.nvim/issues/1576 ?

**Comment by mochaaP** [comment]
`solhint` returned a non-zero exit code when running the generator. I'm not familiar with `solhint`, could you reproduce this on the command line?

**Comment by nshen** [comment, marked as answer]
Thanks @mochaaP , my root_dir wasn't correct. it works now, thanks.
